feature-group-definitions/visual-viewport.dist.yml

```diff
@@ -0,0 +1,29 @@
+# Generated from: visual-viewport.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Visual viewport API
+spec: https://drafts.csswg.org/cssom-view-1/#visualViewport
+status:
+  baseline: high
+  baseline_low_date: 2021-08-10
+  baseline_high_date: 2024-02-10
+  support:
+    chrome: "62"
```

Comment:
This is 62 instead of 61 due to the events in https://developer.mozilla.org/en-US/docs/Web/API/VisualViewport#browser_compatibility, but that was just missing `onresize` and `onscroll` properties. I think this should be an editorial override.